plugins/BEdita/Core/src/Model/Table/UserTokensTable.php

```diff
@@ -127,6 +128,18 @@ public function buildRules(RulesChecker $rules)
         return $rules;
     }
 
+    /**
+     * Return the list of allowed token types merging default with configured.
+     *
+     * @return array
+     */
+    public function getTokenTypes()
+    {
+        $confTypes = (array)Configure::read('UserTokens');
```

Comment:
The `UserTokens` configuration section might include more keys in the future. I'd use `UserTokens.types` instead.

Reply:
yeah, maybe it's better